spec/models/aroc_week_3_spec.rb

```diff
@@ -15,19 +15,20 @@
     expected_result = [@user_2.name, @user_3.name, @user_1.name]
 
     # ----------------------- Using Raw SQL-----------------------
-    users = ActiveRecord::Base.connection.execute("
-      select
-        distinct users.name
-      from users
-        join orders on orders.user_id=users.id
-        join order_items ON order_items.order_id=orders.id
-      where order_items.item_id=#{@item_8.id}
-      ORDER BY users.name")
-    users = users.map {|u| u['name']}
+    # users = ActiveRecord::Base.connection.execute("
+    #   select
+    #     distinct users.name
+    #   from users
+    #     join orders on orders.user_id=users.id
+    #     join order_items ON order_items.order_id=orders.id
+    #   where order_items.item_id = #{@item_8.id}
```

Comment:
It looks like this sql query is expecting users who ordered @item_8. Since you aren't referencing @item_8 in your query, I would guess that something is off here. It's probably a false positive that the test is passing. Think about where @item_8.id could fit into this query.